change/@internal-react-components-a2357b20-610f-4d92-929a-b6507db829f2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Center send button",
```

Comment:
nitty nitty nit: Would be nice to say "vertically center send button in multi-line text box".

The comment should make sense standalone in a CHANGELOG.